{"uuid":"9008fba9-aae6-4ab5-be44-0714174297bd","historyId":"f8f2fb21208f5570c565ec225ced94cb","fullName":"com.codenboxautomationlab.testcases.LoginTest.VerifyThatUserCanLogin","labels":[{"name":"package","value":"com.codenboxautomationlab.testcases.LoginTest"},{"name":"testClass","value":"com.codenboxautomationlab.testcases.LoginTest"},{"name":"testMethod","value":"VerifyThatUserCanLogin"},{"name":"parentSuite","value":"All Test Suite"},{"name":"suite","value":"Login Test"},{"name":"subSuite","value":"com.codenboxautomationlab.testcases.LoginTest"},{"name":"host","value":"DESKTOP-9E9MS69"},{"name":"thread","value":"1840@DESKTOP-9E9MS69.TestNG-test-Login Test-1(52)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"Login Page"},{"name":"feature","value":"Signup Feature"}],"links":[],"name":"Attempt to Login","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [false] but found [true]","trace":"java.lang.AssertionError: expected [false] but found [true]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1578)\r\n\tat org.testng.Assert.assertFalse(Assert.java:79)\r\n\tat org.testng.Assert.assertFalse(Assert.java:89)\r\n\tat com.codenboxautomationlab.testcases.LoginTest.VerifyThatUserCanLogin(LoginTest.java:20)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)\r\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)\r\n\tat org.testng.internal.thread.graph.TestNGFutureTask.run(TestNGFutureTask.java:22)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\r\n\tat java.base/java.lang.Thread.run(Thread.java:1570)\r\n"},"stage":"finished","steps":[{"name":"Load the signup page","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1724669470621,"stop":1724669472055},{"name":"Fill Login form","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1724669472055,"stop":1724669472922},{"name":"Get error message display status","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1724669472923,"stop":1724669473003}],"attachments":[],"parameters":[],"start":1724669470602,"stop":1724669473013}